apis/filesystem/src/main/java/org/jclouds/filesystem/config/FilesystemBlobStoreContextModule.java

```diff
@@ -39,7 +39,10 @@
    protected void configure() {
       bind(BlobStore.class).to(LocalBlobStore.class);
       install(new BlobStoreObjectModule());
-      bind(ConsistencyModel.class).toInstance(ConsistencyModel.STRICT);
+      if (System.getProperty("os.name", "").toLowerCase().contains("windows"))
```

Comment:
Use [filesystem's `TestUtils`](https://github.com/jclouds/jclouds/blob/master/apis/filesystem/src/test/java/org/jclouds/filesystem/utils/TestUtils.java#L224)? Or, better still, move that method to the [core `TestUtils` class](https://github.com/jclouds/jclouds/blob/master/core/src/test/java/org/jclouds/utils/TestUtils.java)?